sealer/cmd/apply.go

```diff
@@ -16,9 +16,9 @@ limitations under the License.
 package cmd
 
 import (
-	"github.com/spf13/cobra"
 	"github.com/alibaba/sealer/apply"
 	"github.com/alibaba/sealer/logger"
+	"github.com/spf13/cobra"
 
 	"os"
```

Comment:
shoud sort the import 
![image](https://user-images.githubusercontent.com/8912557/116776753-d760d680-aa9c-11eb-9b55-229c8c05e31d.png)